pkg/importer/dataStream.go

```diff
@@ -683,12 +691,14 @@ func copy(r io.Reader, out string, qemu bool, imageSize string) error {
 		if err != nil {
 			return errors.Wrap(err, "Local qcow to raw conversion failed")
 		}
-		if imageSize != "" {
-			err = ResizeImage(dest, imageSize)
-		}
-		if err != nil {
-			return errors.Wrap(err, "Resize of image failed")
-		}
+		// Set the dest to out, since we just wrote that
+		dest = out
```

Comment:
This sets it so that once the file is downloaded and converted (non streaming path) that the actual output file is resized instead of the temp file.